istio/metadata.csv

```diff
@@ -196,3 +196,49 @@ istio.galley.validation.cert_key_updates,gauge,,,,Galley validation webhook cert
 istio.galley.validation.config_load,gauge,,,,k8s webhook configuration (re)loads.,0,istio,
 istio.galley.validation.config_update,gauge,,,,k8s webhook configuration updates.,0,istio,
 istio.galley.validation.passed,gauge,,,,Resource is valid.,0,istio,
+istio.citadel.secret_controller.csr_err_count,count,,,,The number of errors occurred when creating the CSR.,-1,istio,
```

Comment:
There is an `error` unit that could fit nicely here.
```suggestion
istio.citadel.secret_controller.csr_err_count,count,,error,,The number of errors occurred when creating the CSR.,-1,istio,
```

Reply:
Done.